src/suricata-plugin.h

```diff
@@ -52,18 +62,13 @@ typedef struct SCCapturePlugin_ {
 
 int SCPluginRegisterCapture(SCCapturePlugin *);
 
-// Every change in the API used by plugins should change this number
-#define SC_PLUGIN_API_VERSION 8
-
```

Comment:
This change conflicts with the nDPI one. 

Seems that we'd need first a `next` version with this, then one with `nDPI` and the necessary update.